pypi_formula_mappings.json

```
@@ -100,6 +100,9 @@
   "scipy": {
     "exclude_packages": ["numpy"]
   },
+  "semgrep": {
+    "package_name": "semgrep"
+  },
```

I think it can be just

```suggestion
  "semgrep": "semgrep",
```

---

Good call.